seatunnel-common/src/main/java/org/apache/seatunnel/common/utils/FileUtils.java

```diff
@@ -80,4 +84,77 @@ public static void createNewFile(String filePath) {
             createParentFile(file);
         }
     }
+
+    /**
+     * return the line number of file
+     *
+     * @param filePath The file need be read
+     * @return
+     */
+    public static Long getFileLineNumber(@NonNull String filePath) {
+        try (LineNumberReader lineNumberReader = new LineNumberReader(new FileReader(filePath))) {
+            lineNumberReader.skip(Long.MAX_VALUE);
```

Comment:
Why we should skip some char?

Reply:
> 

I will use replace LineNumberReader, because if the file content chars more than Long.MAX_VALUE, it will error.

```
try {
        return Files.lines(Paths.get(filePath)).count();
    } catch (IOException e) {
        return -1;
    }
```